logging/logadmin/sinks_test.go

```
@@ -89,40 +111,16 @@ func initSinks(ctx context.Context) func() {
 		}
 	}
 	if integrationTest {
-		for _, bn := range bucketNames(ctx, storageClient) {
-			if bucketIDs.Older(bn, 24*time.Hour) {
```

I'm curious what's been flaky about this if the error is ignored.